lib/v2/nationalgeographic/latest-stories.js

```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const asyncPool = require('tiny-async-pool');
+
+module.exports = async (ctx) => {
+    const currentUrl = 'https://www.nationalgeographic.com/pages/topic/latest-stories';
+    const response = await got(currentUrl);
+    const $ = cheerio.load(response.data);
+    const items = [];
+    for await (const data of asyncPool(5, $('article a[class="AnchorLink PromoTile__Link"]'), (item) => {
+        item = $(item);
+        const link = item.attr('href');
+
+        return ctx.cache.tryGet(link, async () => {
+            const response = await got(link);
+            const $ = cheerio.load(response.data);
+            const article = $('article section.Article__Content');
```

Comment:
Please use the embedded object `__natgeo__` instead.

<details><summary>Screenshot</summary>

![image](https://github.com/DIYgod/RSSHub/assets/11386903/900bb4db-bb56-4dde-867b-7680fa72c78f)
</details>